Ifzan/binary.js

```diff
@@ -0,0 +1,31 @@
+function binary (arr,x,y,find)
+{
+    if(Array.isArray(arr) !== true)
```

Comment:
if(!Array.isArray(arr) || !arr.length)   => this will check for both valid array and non empty array as well

Reply:
ok